### PR TITLE
Refactor some stuff

### PR DESCRIPTION
**Pull request by panleone**
## Abstract
A bit of refactor, I suggest to review commit by commit

- first commit: remove the wallet from the network constructor;
- second commit:  add a `use_network.js` class, intermediate between VUE and .js logic
- third commit: use the new composable class in place of importing the global network class
- fourth commit: clean up unused imports and a function in global.js
- fifth commit: get rid of the global block count in `Activity.js`

---

### CODE REVIEW

File: scripts/composables/use_network.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { ref } from 'vue';
+import { getEventEmitter } from '../event_bus.js';
+import { defineStore } from 'pinia';
+
+export const useNetwork = defineStore('network', () => {
+    const explorerUrl = ref('');
```
Comment (Duddino) [inline scripts/composables/use_network.js:6]: Either watch the variable and change explorer or set is as read only.

**Review by Duddino — COMMENTED** [review]
other than that nit should be good to merge

**Review by JSKitty — CHANGES_REQUESTED** [review]
`openExplorer` was used by the Governance Tab for opening addresses, it'll need replacing with something (until `use_network.js` can be used with Duddino's Vue Governance port)

image

**Comment by panleone** [comment]
> `openExplorer` was used by the Governance Tab for opening addresses, it'll need replacing with something (until `use_network.js` can be used with Duddino's Vue Governance port)
> 
> image

nice catch, I don't know why I thought it was unused

**Review by JSKitty — APPROVED** [review]
**tACK** 1edef671989e37d6e80bf515b4e4161f6de89202

Good little refactor.

**Pull request merged**